docs/disaster-recovery.md

```diff
@@ -0,0 +1,399 @@
+# Failover and Failback In DR
+
+[RBD mirroring](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/) is an asynchronous replication of RBD images between multiple Ceph clusters.
+This capability is available in two modes:
+
+* Journal-based: Every write to the RBD image is first recorded to the associated journal before modifying the actual image. The remote cluster will read from this associated journal and replay the updates to its local image.
+* Snapshot-based: This mode uses periodically scheduled or manually created RBD image mirror-snapshots to replicate crash-consistent RBD images between clusters.
+
+Here, Snapshot based mirroring approach is used to asynchronously replicate RBD images for Disaster Recovery.
+
+## Create Ceph Pools
+
+In this section we create specific ceph pools that are RBD mirroring enabled for use with the DR use case.
+
+>:bulb: **Note:** It is also feasible to edit existing pools and enable them for replication.
+
+Execute the following steps on each peer cluster to create mirror enabled pools:
+
+* Create a Ceph pool that is mirroring enabled, adding the section spec.mirroring in the CephBlockPool CR, example:
```

Comment:
This is not required. please add a link on how to enable pool mirroring to ceph documentation.